hedera-node/src/main/java/com/hedera/services/ledger/PureTransferSemanticChecks.java

```diff
@@ -75,7 +84,7 @@ public ResponseCodeEnum fullPureValidation(
 
 		final var hbarAdjusts = hbarAdjustsWrapper.getAccountAmountsList();
 
-		if (hasRepeatedAccount(hbarAdjusts)) {
+		if (validateRepeatableIds && hasRepeatedAccount(hbarAdjusts)) {
```

Comment:
This is a catastrophic bug, because the current `TransferLogic.doZeroSum()` assumes that the ℏ balance of any account changes **at most once** in its input list. If this invariant is violated, it changes the total ℏ supply. 😰 

We 100% need to refactor make that method safer, but until we do...we shouldn't do this. 😁